src/dmd/dtemplate.d

```diff
@@ -794,6 +794,31 @@ extern (C++) final class TemplateDeclaration : ScopeDsymbol
         return result;
     }
 
+    /******************************
+     * Create a scope for the parameters of the TemplateInstance
+     * ti in the parent scope sc from the ScopeDsymbol paramsym.
```

Comment:
Doc nit: typically backticks are used to highlight variables / symbols (though DMD's codebase isn't a good role model :/